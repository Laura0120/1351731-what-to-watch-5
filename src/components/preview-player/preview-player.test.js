import React from 'react';
import renderer from 'react-test-renderer';

import PreviwPlayer from './preview-player';

it(`PreviwPlayer is rendered correctly`, () => {
  const tree = renderer.create(
      <PreviwPlayer
        video={`http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`}
        preview={`https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/macbeth.jpg`}
        isPlaying={false}
      >
        <video />
      </PreviwPlayer>, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});

