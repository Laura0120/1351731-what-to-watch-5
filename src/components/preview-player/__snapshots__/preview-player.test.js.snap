// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviwPlayer is rendered correctly 1`] = `
<video
  height="175"
  muted={true}
  poster="https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/macbeth.jpg"
  src="http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4"
  width="280"
>
   
</video>
`;
